tests/network/sriov.go

```diff
@@ -702,7 +702,7 @@ func waitVMI(vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error)
 func deleteVMI(vmi *v1.VirtualMachineInstance) error {
 	virtClient := kubevirt.Client()
 
-	GinkgoWriter.Println("sriov:", vmi.Name, "deletion started")
+	GinkgoWriter.Println("sriov:", vmi.Name, "checking virtual machine instance has three interfaces")
```

Comment:
how come the new print replaces the old / related to this PR?
maybe leftover?

Reply:
This change should not be in this commit, its a mistake, I will remove it